geopandas/_vectorized.py

```diff
@@ -83,6 +83,9 @@ def _pygeos_to_shapely(geom):
     if pygeos.is_empty(geom) and pygeos.get_type_id(geom) == 0:
         # empty point does not roundtrip through WKB
         return shapely.wkt.loads("POINT EMPTY")
+    elif pygeos.get_type_id(geom) == 2:
```

Comment:
If it's not already obvious, this doesn't use pygeo.is_ring because that is true for things for linestrings which can be specialised to linearrings. 

I don't think it's possible to do this conversion more directly, but might be missing something.

Reply:
No, this is fine (I don't think there is a better way (shapely unpickling basically does the same, as that has the same issue with WKB)